data/rulesets.ts

```diff
@@ -445,7 +445,7 @@ export const Rulesets: {[k: string]: FormatData} = {
 			this.add('clearpoke');
 			for (const pokemon of this.getAllPokemon()) {
 				const details = pokemon.details.replace(', shiny', '')
-					.replace(/(Arceus|Gourgeist|Pumpkaboo|Xerneas|Silvally|Zacian|Zamazenta|Urshifu)(-[a-zA-Z?-]+)?/g, '$1-*');
```

Comment:
You might not be done with the pr yet, but removing Zacian/Zamazenta from this list would reveal their formes in regular formats like VGC.